portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Adetowo Ademiluyi's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+    <h1 id="myHeader" font-size: 50px>PROJECTS</h1>
+    <p>Open this PDF file to see my resume <a href="new_images/resume.jpg">example</a>.</p>
```

Comment:
Remove "PDF", because the opened file is a jpg file.

Reply:
oh is it possible to edit my code on here? i already fixed the changes on my current code and you'd be able to see them in my next commit.